content/livesync/models/models.textile

```diff
@@ -115,11 +115,11 @@ A merge function is required when you instantiate a model. The merge function in
 The following example is a merge function, invoked for all events received on the channel name specified in the model:
 
 ```[javascript]
-async function merge(state: Post, event: OptimisticEvent | ConfirmedEvent) {
+function merge(state: Post, event: OptimisticEvent | ConfirmedEvent) {
   if (event.name === 'addComment') {
      return {
     ...state,
-    comments: state.comments.concat([event.data]),
+    comments: state.comgit pull ments.concat([event.data]),
```

Comment:
that doesn't look right...? 🤔

Reply:
@snikidev see fixup! 23c067bf370063d2c96dd800421b854ea6624c01